src/components/car/AddCar.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addCar } from "../../redux/cars/carsSlice";
+
+const AddCar = () => {
```

Comment:
- [ ] I cannot add cars:

![image](https://github.com/hmunish/final-capstone-frontend/assets/57421823/e2ddee75-1a31-4a23-bdff-be87ad61a2eb)

It seems I'm not authorized to look at the car details even though I'm logged in.

I didn't test the DELETE functionality because I wasn't able to add cars.

If this is a bug, kindly fix it so your app works as intended.  If this is NOT a bug, it means your app requires special procedures to be run correctly (a `.env` file?, any extra steps required?) if this is the case, kindly add those instructions to your readme so all your users are able to test your app correctly. 👍